### PR TITLE
hotfix: Adding spaship.yaml and favicon to Video Library SPA

### PR DESCRIPTION
**Pull request by ghanlohar**
# Closes/Fixes/Resolves

### 
ONEPLAT-1731
# Explain the feature/fix

- Adding spaship.yaml to video library client

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: video-library-client/spaship.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# SPAship config file
+# generated by @spaship/common.config v0.11.0
+
+name: video-library
+path: /video-library
+single: true
```
Comment (riginoommen) [inline video-library-client/spaship.yaml:6]: include buildir param also

Comment (riginoommen) [reply video-library-client/spaship.yaml:6]:
```suggestion
single: true
buildDir: build
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: video-library-client/spaship.yaml (hunk 1 above)
Comment (ghanlohar) [reply video-library-client/spaship.yaml:6]: Done

**Review by riginoommen — APPROVED** [review]

**Review by akhilmhdh — APPROVED** [review]
LGTM

**Pull request merged**